src/ext/Bal/wixstdba/WixStandardBootstrapperApplication.cpp

```diff
@@ -4786,6 +4884,27 @@ class CWixStandardBootstrapperApplication : public CBalBaseBootstrapperApplicati
         m_pControlUninstallButton = NULL;
         ++pAssignControl;
 
+        pAssignControl->wId = WIXSTDBA_CONTROL_CHECKING_FOR_UPDATES_LABEL;
+        pAssignControl->wzName = L"CheckingForUpdatesLabel";
+        pAssignControl->ppControl = &m_pControlCheckingForUpdatesLabel;
+        pAssignControl->dwAutomaticBehaviorType = dwAutomaticBehaviorType;
```

Comment:
This is conflicting with using the `EnableCondition` attribute on the label in the theme. If wixstdba owns whether it should be shown or not, then the `EnableCondition` shouldn't be there (it won't do anything) and the code above should use `m_pControlCheckingForUpdatesLabel` instead of getting the control through `ThemeControlExistsById`. If the user gets to decide, then this shouldn't be a known control to wixstdba.

Reply:
This label doesn't have any conditions.

Reply:
Sorry, I got it mixed up with ones below but it looks like you figured it out. I was trying to provide a variable in these situations so the user could use a condition instead of making wixstdba responsible for its visibility. That allows the user to provide multiple controls that are conditioned off of it. If we're going to keep it this way, then the code above needs to always change the visibility of the control in case the user doesn't have the default `Visible=no`.

Reply:
The variable's set during Detect, so I didn't want to mess with refreshing the Loading page.